templates/etc/nginx/sites-available/fpm.conf.erb

```diff
@@ -0,0 +1,94 @@
+server {
+  listen <%= @port %>;
+  server_name <%= @server_name %> <% @server_aliases.each do |server_alias| -%><%= server_alias %> <% end -%>;
+  server_name_in_redirect off;
+
+
+  <% if @ssl == true %>
+  ssl_certificate           <%= @ssl_certificate %>;
+  ssl_certificate_key       <%= @ssl_certificate_key %>;
+
+  <% if @ssl_dh_param %>
+  ssl_dhparam <%= @ssl_dh_param %>;
+  <% end %>
+
+
+  ssl_session_cache  shared:SSL:10m;
+  ssl_session_timeout  10m;
+  ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+  ssl_prefer_server_ciphers on;
+  ssl_ciphers AES256+EECDH:AES256+EDH:AES128+EDH:!aNULL;
+  <% end %>
+
+  root /var/www/<%= @name %>/current/public;
```

Comment:
maybe make this the default if `root_path = undef` and use `@root_path` if it is defined.